packages/synapse-react-client/src/components/SynapseSearchPageResults/HasAccessWithText.module.css

```diff
@@ -0,0 +1,71 @@
+/* CSS Module for HasAccessWithText component to match EntityType MUI Chip styling */
+
+.hasAccessChip {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: 32px;
+  border-radius: 16px;
+  padding: 0 12px;
+  margin-left: 8px;
+  font-family: 'DM Sans', 'Helvetica', 'Arial', sans-serif;
```

Comment:
Setting the font should not be necessary (it should be inherited)...or is it? 

If it is not necessary, let's remove it. If it is, let's introduce a new CSS Variable